src/OpenSage.FileFormats.Big/BigArchiveEntry.cs

```diff
@@ -10,6 +10,9 @@ public class BigArchiveEntry
         public BigArchive Archive { get; }
 
         public string FullName { get; }
+        public string Name {
+            get { return FullName.Split('\\')[FullName.Split('\\').Length - 1]; }
```

Comment:
You could probably don't want to split twice here. Better use an intermediate variable

Reply:
Do you mean that?
```csharp
public string Name {
            get {
                int itemIndex = FullName.Split('\\').Length - 1;
                return FullName.Split('\\')[itemIndex];
            }
        }
```

Reply:
```csharp
public string Name {
            get {
                var splitted= FullName.Split('\\');
                return splitted[splitted.Length-1];
            }
        }
```